lecture10/Instances.yml

```
@@ -0,0 +1,93 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters: 
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+  MyIP:
+    Type: String
+  RDSMasterUsername:
+    Type: String
+  RDSMasterUserPassword:
+    Type: String
+
+Resources: 
+# ------------------------------------------------------------#
+#  EC2Instance
+# ------------------------------------------------------------#
+  EC2Instance:
+    Type: AWS::EC2::Instance
+    Properties:
+      KeyName: !Ref KeyName
+      NetworkInterfaces:
+        - AssociatePublicIpAddress: true
+          DeviceIndex: 0
+          SubnetId: !ImportValue RaiseTechSampleAppPublicSubnet1a
+          GroupSet:
+            - !Ref SecGroupForEC2
+      ImageId: ami-04f0680f68f076681
+      InstanceType: t2.micro
+      IamInstanceProfile: !ImportValue RaiseTechSampleAppInstanceProfile
+      Tags:
+        - Key: Name
+          Value: EC2Instance-CF
+  SecGroupForEC2:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: forEC2-CF
+      VpcId: !ImportValue RaiseTechSampleAppVPC
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          CidrIp: !Ref MyIP
+          FromPort: 22
+          ToPort: 22
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          SourceSecurityGroupId: !ImportValue RaiseTechSampleAppSecGroupForALB
+      Tags:
+        - Key: Name
+          Value: SecGroup-EC2-CF
+# ------------------------------------------------------------#
+#  RDSInstance
+# ------------------------------------------------------------#
+  RDSInstance:
+    Type: AWS::RDS::DBInstance
+    Properties:
+      AllocatedStorage: 20 
+      DBInstanceClass: db.t3.micro
+      AvailabilityZone: ap-northeast-1a
+      DBName: forCF
+      DBSubnetGroupName: !ImportValue RaiseTechSampleAppSubnetGroup
+      Engine: mysql
+      EngineVersion: 8.0.34
+      MasterUsername: !Ref RDSMasterUsername
+      MasterUserPassword: !Ref RDSMasterUserPassword
+      MultiAZ: false
+      Port: 3306
+      PubliclyAccessible: false
+      StorageEncrypted: true
+      StorageType: gp2
+      VPCSecurityGroups:
+        - !Ref SecGroupForRDS
+      Tags:
+        - Key: Name
+          Value: RDSInstance-CF
+  SecGroupForRDS:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: forRDS-CF
+      VpcId: !ImportValue RaiseTechSampleAppVPC
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 3306
+          ToPort: 3306
+          SourceSecurityGroupId: !Ref SecGroupForEC2
+      Tags:
+        - Key: Name
+          Value: SecGroup-RDS-CF
+
+Outputs:
+  EC2Instance:
+    Value: !Ref EC2Instance
+    Export:
+      Name: RaiseTechSampleAppEC2

```

パスカルケースに統一するなら下記になるかなと！
```suggestion
      Name: RaiseTechSampleAppEc2
```

---

修正しました！